test/platform/test-platform-execute.ts

```diff
@@ -70,18 +71,20 @@ test('execute-handler-command-success', async t => {
   await registerDevice(app, executeHandler, DEVICE_ID, LOCAL_DEVICE_ID);
   const stubs = extractStubs(app);
 
-  // Prepare the stub to expect the command
+  // Prepare the stub to expect the command.
   stubs.deviceManagerStub.addExpectedCommand(
     validCommand,
     new UdpResponseData(EXECUTE_REQUEST_ID, DEVICE_ID, new UdpResponse())
   );
 
   const executeCommands = createSimpleExecuteCommands(
     DEVICE_ID,
-    'action.devices.commands.OnOff'
+    'actions.devices.commands.OnOff',
+    {on: true},
+    {color: 'red'}
```

Comment:
This seems like a command param and not custom data. Can you adopt something closer to a `foo: 'bar'` like in the documentation examples?

Reply:
Replaced with [documentation example](https://developers.google.com/assistant/smarthome/reference/intent/execute).  Also updated example tests.